agent/tool-scripts/bpftrace

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
+# The following script runs a bpftrace program with the program supplied as argument in pbench-register-tool
+# pbench-register-tool --name=bpftrace -- --script=sample_prog.bt
```

Comment:
Can you indent this 2 or 4 spaces to denote this as an example command?

And do I understand this correctly that the original bpftrace command would look like: `bpftrace --script=sample_prog.bt`?

If that is the case, can you add that to the comment?

Reply:
The original command looks like `bpftrace sample_prog.bt`.

Reply:
Okay, so won't `pbench-register-tool --name=bpftrace -- --script=sample_prog.bt` record the tool to run as, `bpftrace --script=sample_prog.bt`?

Reply:
Nope. `tool_cmd="$tool_bin $script"` Will create the command after parsing, resulting in `bpftrace sample_prog.bt`

Reply:
Ah, right, the tool we are registering is the `pbench/tool-scripts/bpftrace`, which takes the `--script` argument.  That then invokes `/usr/bin/bpftrace sample_prog.bt`.  So let's try to capture that in the comments to help make it clear.

Reply:
Added comments for clarity.